server/content.js

```diff
@@ -2,14 +2,56 @@ const fse = require('fs-extra');
 const fs = require('fs');
 const path = require('path');
 
+const fileExistError = new Error('This slug name is already taken');
+
 const save = (contentType, contentName, data, cb) => {
-  console.log(data);
+  const parsedData = JSON.parse(String(data));
+  const filePath = path.resolve('_content', contentType, contentName + '.json');
+  const newFilePath = `_content\\${contentType}\\${parsedData.slug}.json`;
```

Comment:
different OS would use different slash type. let's use path resolve here to let OS decide what slash to use: `path.resolve('_content', contentType, parsedData.slug + '.json`)`